src/classnames.ts

```diff
@@ -172,8 +172,7 @@ function forward($forwardValue: MaybeString, source: Source): string {
     .join(" ");
 }
 
-function classnames(_variations: unknown, ...inputs: Array<Source | null | undefined>): string {
-  const variations = _variations as RootVariations;
+function classnames(variations: RootVariations, ...inputs: Array<Source | null | undefined>): string {
```

Comment:
I recall having an import and when used the compiler will complain about RootVariations vs VariantsOf @mattvalleycodes 
will be worth checking again.